app/v2/otherCgt/createAmend/CreateAmendOtherCgtService.scala

```diff
@@ -43,6 +43,7 @@ class CreateAmendOtherCgtService @Inject() (connector: CreateAmendOtherCgtConnec
       "INVALID_PAYLOAD"           -> InternalError,
       "INVALID_DISPOSAL_DATE"     -> RuleDisposalDateNotFutureError,
       "INVALID_ACQUISITION_DATE"  -> RuleAcquisitionDateError,
+      "OUTSIDE_AMENDMENT_WINDOW"  -> RuleOutsideAmendmentWindow,
```

Comment:
Although it was previously added could you rename the object generally to end with Error `RuleOutsideAmendmentWindowError`